history/history-network.md

```diff
@@ -183,8 +182,21 @@ HistoricalRootsBlockProof = Container[
     slot: Slot
 ]
 
-BlockHeaderProof = Union[None, HistoricalHashesAccumulatorProof, HistoricalRootsBlockProof]
+# Proof that BeaconBlock root is part of HistoricalRoots and thus canonical
+# For Capella and onwards
+HistoricalSummariesProof = Vector[Bytes32, 13]
+
+# Proof for EL BlockHeader for Capella and onwards
+HistoricalSummariesBlockProof = Container[
+    beaconBlockProof: BeaconBlockProof,
```

Comment:
As mentioned on Discord, this type isn't valid for post Capella Beacon Blocks since the proof length is now 12 elements long.  I'm assuming we should add a new `postCapellaBeaconBlockProof` type that's a vector of 12 elements

Reply:
Another (preferred) idea is to change the type of `BeaconBlockProof` to a `List[Bytes32, 12]`

Reply:
I've made the change to List. I'm thinking however about also changing it to 1 same type for also the version that use historical_roots.